eng/common/scripts/Verify-Links.ps1

```diff
@@ -14,7 +14,11 @@ param (
   # list of http status codes count as broken links. Defaults to 400, 401, 404, SocketError.HostNotFound = 11001, SocketError.NoData = 11004
   [array] $errorStatusCodes = @(400, 401, 404, 11001, 11004),
   # flag to allow resolving relative paths or not
-  [bool] $resolveRelativeLinks = $true
+  [bool] $resolveRelativeLinks = $true,
+  # development repo owner from pr
+  [string] $sourceCommit = "",
+  # development repo branch from pr
+  [string] $sourceRepoUrl = ""
```

Comment:
Instead of taking in the repo url we should take in a link replacement regex. I suggest we default this regex to `(https://github.com/.*/blob/)master(/.*)`. If we want to scope it to just one repo we can pass that in in the template like `($(System.PullRequest.sourceRepositoryUri)/blob/)master(/.*)`. Then to replace all we have to do is `$branchRegex -replace "${1}${sourceCommit}${2}"` (note you need a backtick in front of the 1 and 2 variables to pass them to regex.

Reply:
lets use these names `$branchReplaceRegex` and `$branchReplacementName`

Reply:
Fixed as suggested.